src/connectors/xiami.js

```diff
@@ -1,22 +1,17 @@
 'use strict';
 
-Connector.playerSelector = '#player-main';
+Connector.playerSelector = '.play-bar';
 
-Connector.trackSelector = '#J_trackName';
+Connector.trackSelector = '.music .info .title';
 
-Connector.albumSelector = '.ui-track-current .c3>a';
+Connector.artistSelector = '.music .info .singers';
 
-Connector.artistSelector = '#J_trackInfo>a:nth-child(2)';
+Connector.playButtonSelector = '.main-control .play-btn';
 
-Connector.playButtonSelector = '.play-btn';
+Connector.currentTimeSelector = '.audio-progress .range .bar .handle';
 
-Connector.currentTimeSelector = '#J_positionTime';
+Connector.trackArtSelector = '.music .cover-link .active img';
 
-Connector.trackArtSelector = '#J_playerCoverImg';
-
-Connector.getDuration = () => {
-	let text = Connector.getCurrentTime() ? $('#J_durationTime').text() : '';
-	return Util.stringToSeconds(text);
+Connector.isPlaying = () => {
+	return $('.main-control .play-btn').length !== 0;
 };
-
-Connector.getUniqueID = () => $('.ui-track-current .c1').data('id');
```

Comment:
IS this no longer relevant?

Reply:
I don't know how to use getUniqueID. If you let me know, I'll try to reflect it.
Thank you. :)